### ISSUE TITLE
gov-date-input

### ISSUE THREAD

**Issue by scoch** [CJSCommonPlatform/govuk_single_page_pdk#52]
I can't get the angular sample for the gov-date-input working, it doesn't display any elements on the screen. Has this been implemented yet?

<gov-date-input ng-model="$ctrl.model"></gov-date-input>

**Comment by fabiotisci** [comment]
Yes, it's available. You probably need to update your version on package.json in your project and do npm update. Perhaps you should put it to 'latest' so you constantly get the updates... or you can change it manually every time you need something new. Up to you.

**Comment by scoch** [comment]
Thanks @fabiotisci, we only have it set to what Ben had put it to during the migration to PDK. 

Is it safe to have this constantly pulling in the latest? I am happy to take this to the team if it wont pull in any breaking changes later on.

What dependencies do you recommend having set to latest?

**Comment by fabiotisci** [comment]
uhmm... having it to latest is always a bit risky... we try our best not to break things but mistakes could happen. We are going to include the PDK version somewhere in the docs so people can see if they need to update. Maybe that will solve your problem.

**Comment by scoch** [comment]
That would be very helpful. I'll take it to the team during our retro today anyway.

**Comment by scarlz** [comment]
@scoch We will probably look to move to releasing new features on a beta version in the future to because, like you say, latest is a bit volatile.

Also, regarding the `gov-date-input`, I've just added an optional `date-input-format` attribute that accepts the formatting patterns belonging to the angular date filter, and will output this formatted value to ng-model, in case you find this useful.
